pandas/_libs/reshape.pyi

```diff
@@ -1,6 +1,7 @@
 import numpy as np
 
-import pandas._tying as npt
+from pandas._typing import npt
```

Comment:
could replace
`from pandas._typing import npt`
with
`import numpy.typing as npt`
 in stub files, as I think they will not be executed at run-time - cannot fail even if an old version of numpy is installed.

Reply:
yep, since we don't need to guard the import at run-time for stub files sgtm.